cli/parse_pdfs.py

```diff
@@ -489,17 +512,17 @@ def _get_detectron_model(model: str, device: str) -> Detectron2LayoutModel:
 
 
 def get_model(
-    model: str,
-    ocr_agent: str,
+    model_name: str,
+    ocr_agent_type_name: str,
```

Comment:
I would have 'ocr_agent_name' name here as type_name sounds like the actual name of the type.

Reply:
done